config/vehicle_info.param.yaml

```diff
@@ -1,12 +1,12 @@
 /**:
   ros__parameters:
     wheel_radius: 0.39
-    wheel_width: 0.42
-    wheel_base: 2.74 # between front wheel center and rear wheel center
-    wheel_tread: 1.63 # between left wheel center and right wheel center
+    wheel_width: 0.235
+    wheel_base: 2.79 # between front wheel center and rear wheel center
+    wheel_tread: 1.64 # between left wheel center and right wheel center
     front_overhang: 1.0 # between front wheel center and vehicle front
-    rear_overhang: 1.03 # between rear wheel center and vehicle rear
-    left_overhang: 0.1 # between left wheel center and vehicle left
-    right_overhang: 0.1 # between right wheel center and vehicle right
+    rear_overhang: 1.1 # between rear wheel center and vehicle rear
```

Comment:
Where is this from? :thinking: 
I know it's `4890 - 2790 - 1000 = 1100`, but I'm not sure where is `front_overhang = 1000` written.

Reply:
I couldn't find it as far as I looked.
Since overhang is slightly different depending on the fine type, I adopted the original front_overhang 1.0m.

Reply:
Okay, I understand.